### ISSUE TITLE
Ensure all controls have adequate context

### ISSUE THREAD

**Issue by greatislander** [greatislander/pressbooks-reorderable-multiselect#10]
## Is your feature request related to a problem?

Generic controls like 'Add', 'Move Up', etc. may lack context for non-sighted or partially-sighted users.

## Describe the solution you'd like

Ensure that all controls have an accessible name/ARIA description that fully communicates their context.

## Describe alternative solutions you've considered

Not applicable.

## Additional context or notes

Not applicable.
